File: Java/src/main/java/drawing/shapes/Line.java
package drawing.shapes;

public record Line(int[] start, int[] end) {

    public char[] toJPEG() {
        // Pretend this works.
        return new char[0];
    }

    public char[] toPNG() {
        // Pretend this works.
        return new char[0];
    }
}

File: TypeScript/src/drawing/shapes/line.ts

class Line {
	point1: number[];
	point2: number[];

	constructor(point1: number[], point2: number[]) {
		this.point1 = point1;
		this.point2 = point2;
	}

    public toJPEG(): number[] {
        // Pretend this works.
        return [];
    }

    public toPNG(): number[] {
        // Pretend this works.
        return [];
    }
}

export { Line }